packages/wonder-blocks-core/components/unique-id-provider.md

```diff
@@ -1,9 +1,3 @@
-The `UniqueIDProvider` component is how Wonder Blocks components obtain unique identifiers. This component ensures that server-side rendering and initial client rendering match while allowing the provision of unique identifiers for the client.
```

Comment:
Why move this to the implementation file?  It would be good to codify guidelines around documentation in the Wonder Blocks dev guide.

Reply:
Yes, please leave this in the docs too. I think having the styleguide be a manual of sorts in the dev version is really valuable and this is a component that we want people to know about as it is exported for reuse.